utility/annotate-image/annotate.js

```diff
@@ -114,6 +146,8 @@ module.exports = function(RED) {
                         const bufferOutput = getWritableBuffer();
                         pureimage.encodeJPEGToStream(c,bufferOutput.stream,90).then(() => {
                             msg.payload = bufferOutput.getBuffer();
+                            msg.defaultFontSize = defaultFontSize;
```

Comment:
Why do we want to add this ? Can it be deleted ?

Reply:
forgot it in testing. Will fix and remove this weekend and resubmit pull request.

Reply:
Removed msg.defaultFontSize = defaultFontSize;
also edited documentation as it was misleading about Default behaviour